File: python/case3.py
bilangan = int(input('masukan bilangan'))
# menggunakan modulus 2 jika bialangan nya habis di bagi 2 berati genap
genap = bilangan%2
print(genap)
if genap == 0:
    print("bilangan genap")
else:
    print("bilangan ganjil")

File: golang/case3.go
package main

import "fmt"

func main() {
	bilangan := 101
	genap := bilangan % 2

	if genap == 0 {
		fmt.Println(bilangan, "bilangan genap")
	} else {
		fmt.Println(bilangan, "bilangan ganjil")
	}

}
